### DISCUSSION TITLE
Releases in Trunk Based Development

### DISCUSSION THREAD

**Discussion by seporterfield** [Monstarrrr/rebutify#91] [Ideas & Suggestions]
Trunk based dev suggests two methods for making a release.

## Release branch off main
- provides stability, since "the developers are still streaming their commits into the trunk at full speed"
- where no development happens, instead hotfixes are [cherry-picked](https://www.atlassian.com/git/tutorials/cherry-pick) from main
- gets deleted, not merged back into main

## Release from main (tagged commit)
- has all the benefits of the first option but even faster
- no need to make a new branch
- if needed, create a branch off of the tag for cherry-picked hotfixes

Sources:
1. [Trunk based dev: branch for release](https://trunkbaseddevelopment.com/branch-for-release/)
2. [Quick tips: "you're doing it wrong](https://trunkbaseddevelopment.com/youre-doing-it-wrong/)

**Comment by Monstarrrr** [comment]
Why is it closed ?
